seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -730,21 +730,21 @@
         "Remember to use <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code."
       ],
       "challengeSeed": [
-        "function steamroller(arr) {",
+        "function steamrollArray(arr) {",
```

Comment:
Correct, I didn't think it needed a new function name.  I don't know that it"s "bad" per se to change it, I just didn't think it was needed.